packages/featuregates/bundle/config/values.yaml

```diff
@@ -1,6 +1,6 @@
 #@data/values
 ---
-namespace: tkg-system
+namespace: default
```

Comment:
IIUC, this change reflects our divergence from "TKG". In that case, should we have our own namespace, like "tanzu-runtime" maybe?

Reply:
I was just using this change to test locally. I should revert this. But it's not necessary to have our own namespace. We want product teams to define which namespace this should be installed in and set appropriate RBAC.